app/containers/App.tsx

```diff
@@ -31,6 +31,24 @@ export default function App(props: Props) {
         </Navbar.Group>
 
         <Navbar.Group align={Alignment.RIGHT}>
+          <div className={styles.externalLinks}>
+            <a
+              href="https://github.com/Appsilon/mbaza/discussions"
+              target="_blank"
+              rel="noreferrer"
+              className={styles.link}
+            >
+              Forum
```

Comment:
All user-facing text in the application must be translated. See how `useTranslation()` hook is used throughout the application - we need to use it here as well. You'll need to add fields to `translations.json`. For French just use `TODO`; we'll get translations in bulk near the end of the project.

Reply:
Updated this in the PR :)